onnxruntime/core/providers/openvino/backends/basic_backend.cc

```diff
@@ -131,15 +131,21 @@ bool BasicBackend::ValidateSubgraph(std::map<std::string, std::shared_ptr<ov::No
 void BasicBackend::PopulateConfigValue(ov::AnyMap& device_config) {
   device_config = {};
   // Set inference precision based on device precision for OV backend
-  if (global_context_.precision_str.find("FP16") != std::string::npos && global_context_.device_type == "GPU") {
+  if (global_context_.precision_str.find("FP16") != std::string::npos &&
+      global_context_.device_type == "GPU") {
     device_config.emplace(ov::hint::inference_precision("f16"));
   }
   if (global_context_.precision_str.find("FP32") != std::string::npos) {
     device_config.emplace(ov::hint::inference_precision("f32"));
   }
-  if (global_context_.precision_str.find("ACCURACY") != std::string::npos && global_context_.device_type == "GPU") {
-    device_config.emplace(ov::hint::inference_precision(ov::element::undefined));
-    device_config.emplace(ov::hint::execution_mode(ov::hint::ExecutionMode::ACCURACY));
+  if (global_context_.precision_str.find("ACCURACY") != std::string::npos &&
+      global_context_.device_type == "GPU") {
+    if (global_context_.OpenVINO_Version.at(0) >= 2024 && global_context_.OpenVINO_Version.at(1) >= 1) {
+      device_config.emplace(ov::hint::inference_precision(ov::element::undefined));
+      device_config.emplace(ov::hint::execution_mode(ov::hint::ExecutionMode::ACCURACY));
+    } else {
+      device_config.emplace(ov::hint::inference_precision(global_context_.model_precision));
```

Comment:
We should refrain from setting an inference precision here. This else block is true for all non-GPU non-ACCURACY cases. We handle that above anyway (LN139). Otherwise we are assigning an empty "" for non-GPU cases and that's a problem.